src/cf-base.less

```diff
@@ -417,16 +435,24 @@ a {
 p,
 li,
 dd {
-    // Restrict bottom borders to inline text links ...
-
+    // Restrict underlines to inline text links ...
     a {
-        border-bottom-width: 1px;
+        text-decoration: underline;
+
+        .cssgradients & {
+            background-size: (2px + 2px) 2px;
```

Comment:
Yeah, dot width + spacing. The sum needs to match the sum of the values in the `linear-gradient` declaration. I've converted all of these to variables.